### PR TITLE
Auto scroll

### PR DESCRIPTION
**Pull request by amanTHEBreaker**
Scroll-based features tend to involve some bespoke, but what I did here, I added a Courses next to Learn in index.html, which automatically scrolls to this "Free and Best Courses". 
Also applied same changes to learn.html.

### CODE REVIEW

**Comment by ghulamyazdani** [comment]
I liked that you added courses in navbar which scrolls down to courses section !
I think removing  the scroll down text wasn't required!

File: learn.html [hunk 1]
```diff
@@ -60,14 +63,14 @@
       <h1>Start Learning</h1>
       <div class="drop-container">
         <div class="drop"></div>
-        <h5 class="prompt">Scroll down👇️</h5>
```
Comment (ghulamyazdani) [inline learn.html:63]: This line needs to be added again!

**Comment by ghulamyazdani** [comment]
On the learn page navigation bar you are missing the courses bar instead you added the learn bar twice.  
image
you need to fix that.

**Comment by amanTHEBreaker** [comment]
> On the learn page navigation bar you are missing the courses bar instead you added the learn bar twice.
> image
> you need to fix that.

**Pull request closed without merging**